qiskit_experiments/test/mock_iq_backend.py

```diff
@@ -384,3 +420,232 @@ def run(self, run_input: List[QuantumCircuit], **options) -> FakeJob:
             result["results"].append(run_result)
 
         return FakeJob(self, Result.from_dict(result))
+
+
+class MockIQParallelBackend(MockIQBackend):
+    """A mock backend for testing parallel experiments with IQ data."""
+
+    def __init__(
+        self,
+        experiment_helper: MockIQParallelExperimentHelper = None,
+        rng_seed: int = 0,
+        iq_cluster_centers: Optional[List[Tuple[Tuple[float, float], Tuple[float, float]]]] = None,
+        iq_cluster_width: Optional[List[float]] = None,
+    ):
+        """
+        Initialize the backend.
+        Args:
+            experiment_helper: Parallel experiment helper class that contains
+            helper classes for each experiment.
+            rng_seed: The random seed value.
+            iq_cluster_centers: A list of tuples containing the clusters' centers in the
+            IQ plane.
+            There are different centers for different logical values of the qubit.
+            iq_cluster_width: A list of standard deviation values for the sampling of
+            each qubit.
+        """
+        super().__init__(experiment_helper, rng_seed, iq_cluster_centers, iq_cluster_width)
+
+    @property
+    def experiment_helper(self):
+        """return the 'experiment_helper' attribute"""
+        return self._experiment_helper
+
+    @experiment_helper.setter
+    def experiment_helper(self, value):
+        """
+        Setter for the experiment helper.
+        Args:
+            value(MockIQParallelExperimentHelper): The helper for the backend to use for generating IQ
+             shots.
+
+        Raises:
+            ValueError: Raised if the value to set is not of type `MockIQExperimentHelper`
+        """
+        cls = MockIQParallelExperimentHelper
+        if not isinstance(value, cls):
+            raise ValueError(
+                "The input type is <{}> while the type expected type is <{}>.".format(
+                    str(type(value)), str(cls)
+                )
+            )
```

Comment:
I think the convention is to use f-strings in qiskit experiments.

Reply:
Changed strings to f-strings as suggested.